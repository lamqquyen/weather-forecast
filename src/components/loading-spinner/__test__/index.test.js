import React from 'react'
import LoadingSpinner from '../index'
import TestRenderer from 'react-test-renderer';

describe("LoadingSpinner", () => {
  it('renders correctly with loading is true', () => {
    const tree = TestRenderer.create(
      <LoadingSpinner loading={true}/>
    ).toJSON()

    expect(tree).toMatchSnapshot();
  })


  it('renders correctly with loading is false', () => {
    const tree = TestRenderer.create(
      <LoadingSpinner loading={false}/>
    ).toJSON()

    expect(tree).toBeNull();
  })
})