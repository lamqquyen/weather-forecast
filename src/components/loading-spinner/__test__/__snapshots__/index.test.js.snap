// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner renders correctly with loading is true 1`] = `
<div
  className="loading-spinner__wrapper"
>
  <div
    className="loading-spinner__loader"
  />
</div>
`;
